src/ark/analysis/cell_neighborhood_stats.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pandas as pd
 import xarray as xr
+from tqdm.notebook import tqdm
```

Comment:
instead of specifying a particular implementation of `tqdm` we can use `tqdm.auto` to move that responsibility to `tqdm` itself.